config/environment.js

```diff
@@ -113,11 +108,11 @@ module.exports = function(environment) {
   };
 
   if (environment === "development") {
-    ENV.APP.API_HOST_URL = "http://localhost:3000";
+    ENV.APP.API_HOST_URL = "http://localhost:4000";
```

Comment:
Change of port ? 🤔

Reply:
Yeah, because we have to change it manually in environment.js every time while running stock app. API server runs on 4000 so its better to keep it this way.
Thoughts ?